onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -336,11 +341,15 @@ def run_pytorch(
             cache_dir=cache_dir,
             custom_model_class=model_class,
         )
-        tokenizer = AutoTokenizer.from_pretrained(model_name, cache_dir=cache_dir)
 
-        max_input_size = (
-            tokenizer.max_model_input_sizes[model_name] if model_name in tokenizer.max_model_input_sizes else 1024
-        )
+        if config.model_type == "vit" or config.model_type == "swin":
+            max_input_size = 1024  # Just needs to be greater than sequence_length
```

Comment:
Need not set max_input_size for vit/swin. May just use `if model_type not in ["vit", "swin"]: ...`.